ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -268,6 +294,159 @@ private[state] object Conversions {
       }.toMap,
     )
 
+  def decodeTransactionRejectionEntry(
+      entry: DamlTransactionRejectionEntry
+  ): Option[FinalReason] = {
+    def buildStatus(code: Code, message: String) = {
+      Status.of(
+        code.value,
+        message,
+        Seq.empty,
+      )
+    }
```

Comment:
You can shorten this, and perhaps pull it out of the enclosing function, as it doesn't require the method parameter.

```suggestion
    def buildStatus(code: Code, message: String) =
      Status.of(code.value, message, Seq.empty)
```